Extensions/Gamepads-header.json

```diff
@@ -3,7 +3,7 @@
   "extensionNamespace": "",
   "fullName": "Gamepads (controllers)",
   "name": "Gamepads",
-  "version": "0.0.1",
```

Comment:
That was not intentional and Auto-Updated by the script

Reply:
No worries that's fine!